examples/spellchecker/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
If we're adding samples here, perhaps I should just move all the samples from here:

https://github.com/kennykerr/samples-rs

Reply:
If you're comfortable with that, I'd love that!